### PR TITLE
adios2 2.6.0

### PR DESCRIPTION
**Pull request by DesWurstes**
With GCC 10 fixes

This time, broken by: https://github.com/ornladios/ADIOS2/pull/1996

**Ready for merge.**

### CODE REVIEW

**Comment by ax3l** [comment]
cc @chuckatkins @KyleFromKitware @williamfgc @pnorbert

File: Formula/adios2.rb [hunk 1]
```diff
@@ -24,13 +23,6 @@ class Adios2 < Formula
   depends_on "zeromq"
   uses_from_macos "bzip2"
```
Comment (ax3l) [inline Formula/adios2.rb:25]:
```suggestion

  # macOS 10.13 configuration-time issue detecting float types
  # reference: https://github.com/ornladios/ADIOS2/pull/2305
  patch do
    url "https://github.com/ornladios/ADIOS2/pull/2305.patch?full_index=1"
    sha256 "6d0b84af71d6ccf4cf1cdad5e064cca837d505334316e7e78d18fa30a959666a"
  end
```

**Comment by ax3l** [comment]
@DesWurstes thank you for the PR! Can you please apply the suggestion I posted to check if this already addresses the problem on macOS 10.13?

File: Formula/adios2.rb (hunk 1 above)
Comment (KyleFromKitware) [reply Formula/adios2.rb:25]: +1

Let's wait for ornladios/ADIOS2#2305 to get merged before adding this.

**Comment by DesWurstes** [comment]
Added, let's wait for the CI.

**Comment by ax3l** [comment]
Wuhu, this works and https://github.com/ornladios/ADIOS2/pull/2305 was merged :)

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**